javascript/TabSet.js

```diff
@@ -52,7 +52,7 @@
 				}
 
 				// Switch to the correct tab when AJAX loading completes.
-				$(window).one('ajaxComplete', function () {
+				$(document).ready(function() {
```

Comment:
Mentioned in the description above :)

But it seemed that event was firing either before this code runs, or not at all?